### ISSUE TITLE
Doubt in documentation about changing cost

### ISSUE THREAD

**Issue by rossanag** [valhalla/valhalla-docs#114]
According to the documentation it says:

```
var rr = L.Routing.mapzen('mapzen-xxxxxx', {costing:'auto'});
[...]
bikeButton.onClick: function () {
  rr.route({costing: "bicycle"});
}
```

[https://github.com/mapzen/lrm-mapzen](url)

But I got the error that I have a null value: _route_

I applied the following solution:

```
routeControl.getRouter().options.costing = "auto";
routeControl.route();
```

Found here: [http://gis.stackexchange.com/questions/202432/leaflet-change-routing-vehicle-when-clicked-on-button](url)
where routeControl is the object that holds the lrm control.

The code:

```
function crearControlRouting() {
    routeControl = L.Routing.control({
    routeLine: function (route, options) { return L.Routing.mapzenLine(route, options); },

  plan: L.Routing.plan([], {
                   geocoder: L.Control.Geocoder.nominatim({
                   geocodingQueryParams: { countrycodes: 'uy'}
                 }),
                 reverseWaypoints: true,
                 city:'Montevideo',
                 language: 'sp'
               }),
    lineOptions: {
            styles:
                [
                  {color: 'black', opacity: 0.15, weight: 9}, //sombra
                  {color: 'white', opacity: 0.8, weight: 6}, // Contorno
                  {color: 'red', opacity: 1, weight: 4}] // Centro
       },

    router: L.Routing.mapzen('mapzen-xxxxxx', {
                 costing: 'bicycle',
                directions_options: {
                    language: 'es'
                  }
    }),

    collapsible: true,
    formatter: new L.Routing.mapzenFormatter(),
    summaryTemplate:'<div class="start">{name}</div><div class="info {costing}">{distance}, {time} aproximado</div>'

}).addTo(mapa);

```

Thanks for any clarification:

**Comment by hanbyul-here** [comment]
Was this about this issue? https://github.com/mapzen/lrm-mapzen/issues/65

**Comment by rmglennon** [comment]
It looks like this has been solved in mapzen/lrm-mapzen#65. Please reopen if there are still specific issues.
